/*
 * This file was automatically generated by EvoSuite
 * Sun Mar 22 06:40:49 GMT 2020
 */

package sg.edu.nus.comp.cs4218.exception;

import org.junit.Test;
import static org.junit.Assert.*;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;
import sg.edu.nus.comp.cs4218.exception.CutException;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = true, useJEE = true) 
public class CutException_ESTest extends CutException_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      CutException cutException0 = new CutException("yzQFbdEDg~L3DZOu=");
  }
}
